tests/grf/test_transformations.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+import numpy.testing as npt
```

Comment:
Everywhere else we `import numpy.typing as npt` and access the `testing` functionality as `np.testing.assert_*(...)`.

We should stick to one convention. I don't have an opinion, but given that the rest of the codebase uses the convention described above, it would be easier to fix things in this PR.